flink-streaming-java/src/main/java/org/apache/flink/streaming/api/transformations/SinkV1Adapter.java

```diff
@@ -447,5 +451,13 @@ public void commit(Collection<CommitRequest<CommT>> committables)
                 committables.forEach(CommitRequest::retryLater);
```

Comment:
Here seems should be `failures.forEach`?

Reply:
This is not easily possible because only committable are retriable at this point. I would need to add a special logic when this happens. I rely here on the fact that I assume all `commit` calls are idempotent.

Reply:
Thanks for the clarification! Might we add some comments for this~?

Reply:
Good idea, I add a comment.